### ISSUE TITLE
Create bsb theme

### ISSUE THREAD

**Issue by saschatimme** [BuckleTypes/bs-react-native#35]
Now that Bucklescript supports themes / templates, we should pitch one for React Native :)

**Comment by saschatimme** [comment]
Example for Reason React https://github.com/bloomberg/bucklescript/pull/1696

**Comment by wokalski** [comment]
cc @brentvante maybe bsb theme is even better (i.e. more lightweight) than the scripts approach.

**Comment by grabbou** [comment]
What is the status here? We opted for sticking to `react-native init --template rebolt` (can be renamed to `reason` now) in order to stay as close to React Native userland as possible.

**Comment by wokalski** [comment]
Fair point. `react-native init --template reason` makes sense. I hope it's not too bloated though (?)

**Comment by grabbou** [comment]
Tried to keep it as minimal as possible. I guess it makes sense to move it
somewhere now that it's `bs-react-native`, not `rebolt` generator.

Need to figure out how and where to move it - do you think it makes sense
for it to stay in a separate repository? Or here, as a folder? Or use Yarn
workspaces?

>

**Comment by wokalski** [comment]
I think it's fine to put it here as a folder. How do we make `react-native` cli tooling aware of it though?

**Comment by grabbou** [comment]
Has to be published as a separate package to npm registry with
`react-native-template-xx` where `xx` is what you supply as a template flag.

**Comment by wokalski** [comment]
Cool, let's do it then. I'm going to rename this issue.

**Comment by grabbou** [comment]
Going to move this package https://github.com/callstackincubator/rebolt-generator here as a PR and we can release it under a new name.

There are some limitations that we are going to work on as a part of ongoing CLI development. For now, this is the best we can get.
